### DISCUSSION TITLE
(failing to) get started with motionplus

### DISCUSSION THREAD

**Discussion by pootle** [Motion-Project/motionplus#182] [Q&A] [answered]
Used motioneye many years ago, now trying to get started with motionplus, but cannot get it to pick up the raspberry pi camera

Clean build of rpiOS (64bit) lite on a rpi 3+
installed motionplus with 
`sudo dpkg -i bookworm_motionplus_0.2.1-1_arm64.deb`
then fixed up dependencies with
`sudo apt-get -f install`

checked camera working with libcamera-still
added samba support to access files from windoze
modded config file to enable network access and set params to  2, pointed config file at a folder with camera config file, but it fails to 
pickup the camera.
Changed the config file to point straight at camera1.conf as well (this version in this post)
I have also tried putting the camera straight into the main config file, and it does appear in the web page, but without the proper params.
What am I doing wrong?

on web site, Configuration - Camera config_dir field is blank and camera field contains the location of the main config file.

here is the camera1.conf file:
`; projects/motionconf/camera1.conf
;
; This config file was generated by motionplus 0.2.1

;*************************************************
; Configuration options specific to camera 1
;*************************************************

;*************************************************
;*****   Camera
;*************************************************
device_name MyCam1
device_id 101

;*************************************************
;*****   Source
;*************************************************
libcam_device camera0

;*************************************************
;*****   Image
;*************************************************
width 800
height 600
framerate 15

;*************************************************
;*****   Overlays
;*************************************************
text_left CAMERA 1

;*************************************************
;*****   Movie
;*************************************************
movie_filename CAM01_%t-%v-%Y%m%d%H%M%S`

and here is the main config file. I have checked the device config file really points at the file above (and even set permissions to 666), but the log says no camera configured:
`; This config file was generated by motionplus 0.2.1
; Documentation:  /usr/share/doc/motionplus/motionplus_guide.html
;
; This file contains only the basic configuration options to get a
; system working.  There are many more options available.  Please
; consult the documentation for the complete list of all options.
;

;*************************************************
;*****   System
;*************************************************
daemon off
setup_mode off
log_file /var/log/motionplus/mp.log
log_level 9
log_type ALL

;*************************************************
;*****   Camera
;*************************************************
;device_name
;device_id
;target_dir

;*************************************************
;*****   Source
;*************************************************
;v4l2_device /dev/video0
;netcam_url

;*************************************************
;*****   Image
;*************************************************
;width 640
;height 480
;framerate 15

;*************************************************
;*****   Overlays
;*************************************************
text_right %Y-%m-%d\n%T

;*************************************************
;*****   Method
;*************************************************
emulate_motion off
threshold 1500

;*************************************************
;*****   Masks
;*************************************************
noise_level 32
despeckle_filter EedDl

;*************************************************
;*****   Detection
;*************************************************
minimum_motion_frames 1
event_gap 60
pre_capture 3
post_capture 10

;*************************************************
;*****   Scripts
;*************************************************
on_event_start
on_event_end

;*************************************************
;*****   Picture
;*************************************************
picture_output off
picture_filename %v-%Y%m%d%H%M%S-%q

;*************************************************
;*****   Movie
;*************************************************
movie_output on
movie_max_time 120
movie_quality 45
movie_container mkv
movie_filename %v-%Y%m%d%H%M%S

;*************************************************
;*****   Web Control
;*************************************************
webcontrol_port 8080
webcontrol_localhost off
webcontrol_parms 2

;*************************************************
;*****   Web Stream
;*************************************************
stream_preview_scale 25
stream_preview_method mjpg

;*************************************************
; Device config files - One for each device.
;*************************************************
camera /home/iain/projects/motionconf/camera1.conf
; camera /usr/etc/motionplus/camera2.conf
; camera /usr/etc/motionplus/camera3.conf
; sound /usr/etc/motionplus/sound1.conf

;*************************************************
; Directory to read '.conf' files for devices.
;*************************************************
; config_dir /usr/etc/motionplus/conf.d
`

Here is the tail of the log file:
`Aug 31 12:32:54 [INF][ALL][00:motionplus] snd_params
Aug 31 12:32:54 [INF][ALL][00:motionplus] snd_window                hamming
Aug 31 12:32:54 [INF][ALL][00:motionplus] snd_show                  off
Aug 31 12:32:54 [DBG][ALL][00:motionplus] motpls_ntc: v4l2   : available
Aug 31 12:32:54 [DBG][ALL][00:motionplus] motpls_ntc: webp   : available
Aug 31 12:32:54 [DBG][ALL][00:motionplus] motpls_ntc: libcam : available
Aug 31 12:32:54 [DBG][ALL][00:motionplus] motpls_ntc: mysql  : not available
Aug 31 12:32:54 [DBG][ALL][00:motionplus] motpls_ntc: MariaDB: available
Aug 31 12:32:54 [DBG][ALL][00:motionplus] motpls_ntc: sqlite3: available
Aug 31 12:32:54 [DBG][ALL][00:motionplus] motpls_ntc: pgsql  : available
Aug 31 12:32:54 [DBG][ALL][00:motionplus] motpls_ntc: nls    : available
Aug 31 12:32:54 [DBG][ALL][00:motionplus] motpls_ntc: alsa   : available
Aug 31 12:32:54 [DBG][ALL][00:motionplus] motpls_ntc: fftw3  : available
Aug 31 12:32:54 [NTC][STR][00:motionplus] webu_init_webcontrol: Starting webcontrol on port 8080
Aug 31 12:32:54 [INF][ALL][00:motionplus] util_parms_add: Parsed: >event< >on<
Aug 31 12:32:54 [INF][ALL][00:motionplus] util_parms_add: Parsed: >snapshot< >on<
Aug 31 12:32:54 [INF][ALL][00:motionplus] util_parms_add: Parsed: >pause< >on<
Aug 31 12:32:54 [INF][ALL][00:motionplus] util_parms_add: Parsed: >restart< >on<
Aug 31 12:32:54 [INF][ALL][00:motionplus] util_parms_add: Parsed: >stop< >on<
Aug 31 12:32:54 [INF][ALL][00:motionplus] util_parms_add: Parsed: >config_write< >on<
Aug 31 12:32:54 [INF][ALL][00:motionplus] util_parms_add: Parsed: >camera_add< >on<
Aug 31 12:32:54 [INF][ALL][00:motionplus] util_parms_add: Parsed: >camera_delete< >on<
Aug 31 12:32:54 [INF][ALL][00:motionplus] util_parms_add: Parsed: >config< >on<
Aug 31 12:32:54 [INF][ALL][00:motionplus] util_parms_add: Parsed: >ptz< >on<
Aug 31 12:32:54 [INF][ALL][00:motionplus] util_parms_add: Parsed: >movies< >on<
Aug 31 12:32:54 [INF][ALL][00:motionplus] util_parms_add: Parsed: >action_user< >on<
Aug 31 12:32:54 [DBG][STR][00:motionplus] webu_mhd_features_basic: Basic authentication: available
Aug 31 12:32:54 [DBG][STR][00:motionplus] webu_mhd_features_digest: Digest authentication: available
Aug 31 12:32:54 [DBG][STR][00:motionplus] webu_mhd_features_ipv6: IPV6: available
Aug 31 12:32:54 [DBG][STR][00:motionplus] webu_mhd_features_tls: SSL/TLS: available
Aug 31 12:32:54 [NTC][STR][00:motionplus] webu_init_webcontrol: Started webcontrol on port 8080
Aug 31 12:32:54 [NTC][ENC][00:motionplus] motpls_av_init: libavcodec  version 59.37.100
Aug 31 12:32:54 [NTC][ENC][00:motionplus] motpls_av_init: libavformat version 59.27.100
Aug 31 12:32:54 [NTC][ALL][00:motionplus] main: Motionplus pid: 1277
Aug 31 12:32:54 [NTC][ALL][00:motionplus] main: No camera or sound configuration files specified.
Aug 31 12:32:54 [NTC][ALL][00:motionplus] main: Waiting for camera or sound configuration to be added via web control.
Aug 31 12:35:49 [DBG][STR][00:wc00] webu_parseurl: Sent url: /
Aug 31 12:35:49 [DBG][STR][00:wc00] webu_parseurl: Decoded url: /
Aug 31 12:35:49 [DBG][STR][00:wc00] webu_parms_edit: camid: >< thread: >-1< cmd1: >< cmd2: >< cmd3: ><
Aug 31 12:35:49 [DBG][ALL][00:wc00] webu_hostname: Full Host:  //192.168.33.43:8080
Aug 31 12:35:49 [INF][ALL][00:wc00] webu_client_connect: Connection from: 192.168.33.34
Aug 31 12:35:49 [DBG][ALL][00:wc00] webu_hostname: Full Host:  //192.168.33.43:8080
Aug 31 12:35:49 [DBG][STR][00:wc00] webu_answer_get: processing get
Aug 31 12:35:49 [INF][STR][00:wc00] webu_mhd_deinit: Closing connection
Aug 31 12:35:49 [DBG][STR][00:wc00] webu_parseurl: Sent url: /0/config.json
Aug 31 12:35:49 [DBG][STR][00:wc00] webu_parseurl: Decoded url: /0/config.json
Aug 31 12:35:49 [DBG][STR][00:wc00] webu_parms_edit: camid: >0< thread: >-1< cmd1: >config.json< cmd2: >< cmd3: ><
Aug 31 12:35:49 [DBG][ALL][00:wc00] webu_hostname: Full Host:  //192.168.33.43:8080
Aug 31 12:35:49 [DBG][ALL][00:wc00] webu_hostname: Above message repeats 1 times
Aug 31 12:35:49 [DBG][STR][00:wc00] webu_answer_get: processing get`

**Comment by Mr-Dave** [comment, marked as answer]
Probably want to format the tail of the log output and also provide the full log.  (Which I suspect will indicate that you didn't specify the camera configuration file path correctly)

**Reply by pootle** [reply]
A bad config filename appears to fail silently - even with loglevel 9 there is nothing in the log. I did check the filename.

I suspect a permissions problem. I moved the camera config to /etc/motionplus and it now works OK.

Sorry about the tail - here is the log from a failed run - the camera line just disappears without trace:

`Aug 31 12:32:54 [NTC][ALL][00:motionplus] log_init: MotionPlus 0.2.1 started
Aug 31 12:32:54 [NTC][ALL][00:motionplus] log_init: Using log type (ALL) log level (ALL)
Aug 31 12:32:54 [NTC][ALL][00:motionplus] mytranslate_init: Language: English
Aug 31 12:32:54 [INF][ALL][00:motionplus] conf_parms_log: Logging configuration parameters from all files
Aug 31 12:32:54 [INF][ALL][00:motionplus] Config file: /etc/motionplus/motionplus.conf
Aug 31 12:32:54 [INF][ALL][00:motionplus] daemon                    off
Aug 31 12:32:54 [INF][ALL][00:motionplus] setup_mode                off
Aug 31 12:32:54 [INF][ALL][00:motionplus] pid_file                  
Aug 31 12:32:54 [INF][ALL][00:motionplus] log_file                  /var/log/motionplus/mp.log
Aug 31 12:32:54 [INF][ALL][00:motionplus] log_level                 9
Aug 31 12:32:54 [INF][ALL][00:motionplus] log_type                  ALL
Aug 31 12:32:54 [INF][ALL][00:motionplus] native_language           on
Aug 31 12:32:54 [INF][ALL][00:motionplus] device_name               
Aug 31 12:32:54 [INF][ALL][00:motionplus] device_id                 0
Aug 31 12:32:54 [INF][ALL][00:motionplus] device_tmo                30
Aug 31 12:32:54 [INF][ALL][00:motionplus] pause                     off
Aug 31 12:32:54 [INF][ALL][00:motionplus] target_dir                .
Aug 31 12:32:54 [INF][ALL][00:motionplus] watchdog_tmo              30
Aug 31 12:32:54 [INF][ALL][00:motionplus] watchdog_kill             10
Aug 31 12:32:54 [INF][ALL][00:motionplus] v4l2_device               
Aug 31 12:32:54 [INF][ALL][00:motionplus] v4l2_params               
Aug 31 12:32:54 [INF][ALL][00:motionplus] netcam_url                <redacted>
Aug 31 12:32:54 [INF][ALL][00:motionplus] netcam_params             
Aug 31 12:32:54 [INF][ALL][00:motionplus] netcam_high_url           <redacted>
Aug 31 12:32:54 [INF][ALL][00:motionplus] netcam_high_params        
Aug 31 12:32:54 [INF][ALL][00:motionplus] netcam_userpass           <redacted>
Aug 31 12:32:54 [INF][ALL][00:motionplus] libcam_device             
Aug 31 12:32:54 [INF][ALL][00:motionplus] libcam_params             
Aug 31 12:32:54 [INF][ALL][00:motionplus] width                     640
Aug 31 12:32:54 [INF][ALL][00:motionplus] height                    480
Aug 31 12:32:54 [INF][ALL][00:motionplus] framerate                 15
Aug 31 12:32:54 [INF][ALL][00:motionplus] rotate                    0
Aug 31 12:32:54 [INF][ALL][00:motionplus] flip_axis                 none
Aug 31 12:32:54 [INF][ALL][00:motionplus] locate_motion_mode        off
Aug 31 12:32:54 [INF][ALL][00:motionplus] locate_motion_style       box
Aug 31 12:32:54 [INF][ALL][00:motionplus] text_left                 
Aug 31 12:32:54 [INF][ALL][00:motionplus] text_right                %Y-%m-%d\n%T
Aug 31 12:32:54 [INF][ALL][00:motionplus] text_changes              off
Aug 31 12:32:54 [INF][ALL][00:motionplus] text_scale                1
Aug 31 12:32:54 [INF][ALL][00:motionplus] text_event                %Y%m%d%H%M%S
Aug 31 12:32:54 [INF][ALL][00:motionplus] emulate_motion            off
Aug 31 12:32:54 [INF][ALL][00:motionplus] threshold                 1500
Aug 31 12:32:54 [INF][ALL][00:motionplus] threshold_maximum         0
Aug 31 12:32:54 [INF][ALL][00:motionplus] threshold_sdevx           0
Aug 31 12:32:54 [INF][ALL][00:motionplus] threshold_sdevy           0
Aug 31 12:32:54 [INF][ALL][00:motionplus] threshold_sdevxy          0
Aug 31 12:32:54 [INF][ALL][00:motionplus] threshold_ratio           0
Aug 31 12:32:54 [INF][ALL][00:motionplus] threshold_ratio_change    64
Aug 31 12:32:54 [INF][ALL][00:motionplus] threshold_tune            off
Aug 31 12:32:54 [INF][ALL][00:motionplus] secondary_method          none
Aug 31 12:32:54 [INF][ALL][00:motionplus] secondary_params          
Aug 31 12:32:54 [INF][ALL][00:motionplus] noise_level               32
Aug 31 12:32:54 [INF][ALL][00:motionplus] noise_tune                on
Aug 31 12:32:54 [INF][ALL][00:motionplus] despeckle_filter          EedDl
Aug 31 12:32:54 [INF][ALL][00:motionplus] area_detect               
Aug 31 12:32:54 [INF][ALL][00:motionplus] mask_file                 
Aug 31 12:32:54 [INF][ALL][00:motionplus] mask_privacy              
Aug 31 12:32:54 [INF][ALL][00:motionplus] smart_mask_speed          0
Aug 31 12:32:54 [INF][ALL][00:motionplus] lightswitch_percent       0
Aug 31 12:32:54 [INF][ALL][00:motionplus] lightswitch_frames        5
Aug 31 12:32:54 [INF][ALL][00:motionplus] minimum_motion_frames     1
Aug 31 12:32:54 [INF][ALL][00:motionplus] static_object_time        10
Aug 31 12:32:54 [INF][ALL][00:motionplus] event_gap                 60
Aug 31 12:32:54 [INF][ALL][00:motionplus] pre_capture               3
Aug 31 12:32:54 [INF][ALL][00:motionplus] post_capture              10
Aug 31 12:32:54 [INF][ALL][00:motionplus] on_event_start            
Aug 31 12:32:54 [INF][ALL][00:motionplus] on_event_end              
Aug 31 12:32:54 [INF][ALL][00:motionplus] on_picture_save           
Aug 31 12:32:54 [INF][ALL][00:motionplus] on_area_detected          
Aug 31 12:32:54 [INF][ALL][00:motionplus] on_motion_detected        
Aug 31 12:32:54 [INF][ALL][00:motionplus] on_movie_start            
Aug 31 12:32:54 [INF][ALL][00:motionplus] on_movie_end              
Aug 31 12:32:54 [INF][ALL][00:motionplus] on_camera_lost            
Aug 31 12:32:54 [INF][ALL][00:motionplus] on_camera_found           
Aug 31 12:32:54 [INF][ALL][00:motionplus] on_secondary_detect       
Aug 31 12:32:54 [INF][ALL][00:motionplus] on_action_user            
Aug 31 12:32:54 [INF][ALL][00:motionplus] on_sound_alert            
Aug 31 12:32:54 [INF][ALL][00:motionplus] picture_output            off
Aug 31 12:32:54 [INF][ALL][00:motionplus] picture_output_motion     off
Aug 31 12:32:54 [INF][ALL][00:motionplus] picture_type              jpg
Aug 31 12:32:54 [INF][ALL][00:motionplus] picture_quality           75
Aug 31 12:32:54 [INF][ALL][00:motionplus] picture_exif              
Aug 31 12:32:54 [INF][ALL][00:motionplus] picture_filename          %v-%Y%m%d%H%M%S-%q
Aug 31 12:32:54 [INF][ALL][00:motionplus] snapshot_interval         0
Aug 31 12:32:54 [INF][ALL][00:motionplus] snapshot_filename         %v-%Y%m%d%H%M%S-snapshot
Aug 31 12:32:54 [INF][ALL][00:motionplus] movie_output              on
Aug 31 12:32:54 [INF][ALL][00:motionplus] movie_output_motion       off
Aug 31 12:32:54 [INF][ALL][00:motionplus] movie_max_time            120
Aug 31 12:32:54 [INF][ALL][00:motionplus] movie_bps                 400000
Aug 31 12:32:54 [INF][ALL][00:motionplus] movie_quality             45
Aug 31 12:32:54 [INF][ALL][00:motionplus] movie_container           mkv
Aug 31 12:32:54 [INF][ALL][00:motionplus] movie_passthrough         off
Aug 31 12:32:54 [INF][ALL][00:motionplus] movie_filename            %v-%Y%m%d%H%M%S
Aug 31 12:32:54 [INF][ALL][00:motionplus] movie_retain              all
Aug 31 12:32:54 [INF][ALL][00:motionplus] movie_extpipe_use         off
Aug 31 12:32:54 [INF][ALL][00:motionplus] movie_extpipe             
Aug 31 12:32:54 [INF][ALL][00:motionplus] timelapse_interval        0
Aug 31 12:32:54 [INF][ALL][00:motionplus] timelapse_mode            daily
Aug 31 12:32:54 [INF][ALL][00:motionplus] timelapse_fps             30
Aug 31 12:32:54 [INF][ALL][00:motionplus] timelapse_container       mpg
Aug 31 12:32:54 [INF][ALL][00:motionplus] timelapse_filename        %Y%m%d-timelapse
Aug 31 12:32:54 [INF][ALL][00:motionplus] video_pipe                
Aug 31 12:32:54 [INF][ALL][00:motionplus] video_pipe_motion         
Aug 31 12:32:54 [INF][ALL][00:motionplus] webcontrol_port           8080
Aug 31 12:32:54 [INF][ALL][00:motionplus] webcontrol_port2          0
Aug 31 12:32:54 [INF][ALL][00:motionplus] webcontrol_base_path      
Aug 31 12:32:54 [INF][ALL][00:motionplus] webcontrol_ipv6           off
Aug 31 12:32:54 [INF][ALL][00:motionplus] webcontrol_localhost      off
Aug 31 12:32:54 [INF][ALL][00:motionplus] webcontrol_parms          2
Aug 31 12:32:54 [INF][ALL][00:motionplus] webcontrol_interface      default
Aug 31 12:32:54 [INF][ALL][00:motionplus] webcontrol_auth_method    none
Aug 31 12:32:54 [INF][ALL][00:motionplus] webcontrol_authentication <redacted>
Aug 31 12:32:54 [INF][ALL][00:motionplus] webcontrol_tls            off
Aug 31 12:32:54 [INF][ALL][00:motionplus] webcontrol_cert           <redacted>
Aug 31 12:32:54 [INF][ALL][00:motionplus] webcontrol_key            <redacted>
Aug 31 12:32:54 [INF][ALL][00:motionplus] webcontrol_headers        
Aug 31 12:32:54 [INF][ALL][00:motionplus] webcontrol_html           
Aug 31 12:32:54 [INF][ALL][00:motionplus] webcontrol_actions        
Aug 31 12:32:54 [INF][ALL][00:motionplus] webcontrol_lock_minutes   10
Aug 31 12:32:54 [INF][ALL][00:motionplus] webcontrol_lock_attempts  3
Aug 31 12:32:54 [INF][ALL][00:motionplus] webcontrol_lock_script    
Aug 31 12:32:54 [INF][ALL][00:motionplus] stream_preview_scale      25
Aug 31 12:32:54 [INF][ALL][00:motionplus] stream_preview_newline    off
Aug 31 12:32:54 [INF][ALL][00:motionplus] stream_preview_method     mjpg
Aug 31 12:32:54 [INF][ALL][00:motionplus] stream_preview_ptz        on
Aug 31 12:32:54 [INF][ALL][00:motionplus] stream_quality            50
Aug 31 12:32:54 [INF][ALL][00:motionplus] stream_grey               off
Aug 31 12:32:54 [INF][ALL][00:motionplus] stream_motion             off
Aug 31 12:32:54 [INF][ALL][00:motionplus] stream_maxrate            1
Aug 31 12:32:54 [INF][ALL][00:motionplus] stream_scan_time          5
Aug 31 12:32:54 [INF][ALL][00:motionplus] stream_scan_scale         25
Aug 31 12:32:54 [INF][ALL][00:motionplus] database_type             
Aug 31 12:32:54 [INF][ALL][00:motionplus] database_dbname           
Aug 31 12:32:54 [INF][ALL][00:motionplus] database_host             
Aug 31 12:32:54 [INF][ALL][00:motionplus] database_port             0
Aug 31 12:32:54 [INF][ALL][00:motionplus] database_user             <redacted>
Aug 31 12:32:54 [INF][ALL][00:motionplus] database_password         <redacted>
Aug 31 12:32:54 [INF][ALL][00:motionplus] database_busy_timeout     0
Aug 31 12:32:54 [INF][ALL][00:motionplus] sql_event_start           
Aug 31 12:32:54 [INF][ALL][00:motionplus] sql_event_end             
Aug 31 12:32:54 [INF][ALL][00:motionplus] sql_movie_start           
Aug 31 12:32:54 [INF][ALL][00:motionplus] sql_movie_end             
Aug 31 12:32:54 [INF][ALL][00:motionplus] sql_pic_save              
Aug 31 12:32:54 [INF][ALL][00:motionplus] ptz_auto_track            off
Aug 31 12:32:54 [INF][ALL][00:motionplus] ptz_wait                  0
Aug 31 12:32:54 [INF][ALL][00:motionplus] ptz_move_track            
Aug 31 12:32:54 [INF][ALL][00:motionplus] ptz_pan_left              
Aug 31 12:32:54 [INF][ALL][00:motionplus] ptz_pan_right             
Aug 31 12:32:54 [INF][ALL][00:motionplus] ptz_tilt_up               
Aug 31 12:32:54 [INF][ALL][00:motionplus] ptz_tilt_down             
Aug 31 12:32:54 [INF][ALL][00:motionplus] ptz_zoom_in               
Aug 31 12:32:54 [INF][ALL][00:motionplus] ptz_zoom_out              
Aug 31 12:32:54 [INF][ALL][00:motionplus] snd_device                
Aug 31 12:32:54 [INF][ALL][00:motionplus] snd_params                
Aug 31 12:32:54 [INF][ALL][00:motionplus] snd_window                hamming
Aug 31 12:32:54 [INF][ALL][00:motionplus] snd_show                  off
Aug 31 12:32:54 [DBG][ALL][00:motionplus] motpls_ntc: v4l2   : available
Aug 31 12:32:54 [DBG][ALL][00:motionplus] motpls_ntc: webp   : available
Aug 31 12:32:54 [DBG][ALL][00:motionplus] motpls_ntc: libcam : available
Aug 31 12:32:54 [DBG][ALL][00:motionplus] motpls_ntc: mysql  : not available
Aug 31 12:32:54 [DBG][ALL][00:motionplus] motpls_ntc: MariaDB: available
Aug 31 12:32:54 [DBG][ALL][00:motionplus] motpls_ntc: sqlite3: available
Aug 31 12:32:54 [DBG][ALL][00:motionplus] motpls_ntc: pgsql  : available
Aug 31 12:32:54 [DBG][ALL][00:motionplus] motpls_ntc: nls    : available
Aug 31 12:32:54 [DBG][ALL][00:motionplus] motpls_ntc: alsa   : available
Aug 31 12:32:54 [DBG][ALL][00:motionplus] motpls_ntc: fftw3  : available
Aug 31 12:32:54 [NTC][STR][00:motionplus] webu_init_webcontrol: Starting webcontrol on port 8080
Aug 31 12:32:54 [INF][ALL][00:motionplus] util_parms_add: Parsed: >event< >on<
Aug 31 12:32:54 [INF][ALL][00:motionplus] util_parms_add: Parsed: >snapshot< >on<
Aug 31 12:32:54 [INF][ALL][00:motionplus] util_parms_add: Parsed: >pause< >on<
Aug 31 12:32:54 [INF][ALL][00:motionplus] util_parms_add: Parsed: >restart< >on<
Aug 31 12:32:54 [INF][ALL][00:motionplus] util_parms_add: Parsed: >stop< >on<
Aug 31 12:32:54 [INF][ALL][00:motionplus] util_parms_add: Parsed: >config_write< >on<
Aug 31 12:32:54 [INF][ALL][00:motionplus] util_parms_add: Parsed: >camera_add< >on<
Aug 31 12:32:54 [INF][ALL][00:motionplus] util_parms_add: Parsed: >camera_delete< >on<
Aug 31 12:32:54 [INF][ALL][00:motionplus] util_parms_add: Parsed: >config< >on<
Aug 31 12:32:54 [INF][ALL][00:motionplus] util_parms_add: Parsed: >ptz< >on<
Aug 31 12:32:54 [INF][ALL][00:motionplus] util_parms_add: Parsed: >movies< >on<
Aug 31 12:32:54 [INF][ALL][00:motionplus] util_parms_add: Parsed: >action_user< >on<
Aug 31 12:32:54 [DBG][STR][00:motionplus] webu_mhd_features_basic: Basic authentication: available
Aug 31 12:32:54 [DBG][STR][00:motionplus] webu_mhd_features_digest: Digest authentication: available
Aug 31 12:32:54 [DBG][STR][00:motionplus] webu_mhd_features_ipv6: IPV6: available
Aug 31 12:32:54 [DBG][STR][00:motionplus] webu_mhd_features_tls: SSL/TLS: available
Aug 31 12:32:54 [NTC][STR][00:motionplus] webu_init_webcontrol: Started webcontrol on port 8080
Aug 31 12:32:54 [NTC][ENC][00:motionplus] motpls_av_init: libavcodec  version 59.37.100
Aug 31 12:32:54 [NTC][ENC][00:motionplus] motpls_av_init: libavformat version 59.27.100
Aug 31 12:32:54 [NTC][ALL][00:motionplus] main: Motionplus pid: 1277
Aug 31 12:32:54 [NTC][ALL][00:motionplus] main: No camera or sound configuration files specified.
Aug 31 12:32:54 [NTC][ALL][00:motionplus] main: Waiting for camera or sound configuration to be added via web control.
Aug 31 12:35:49 [DBG][STR][00:wc00] webu_parseurl: Sent url: /
Aug 31 12:35:49 [DBG][STR][00:wc00] webu_parseurl: Decoded url: /
Aug 31 12:35:49 [DBG][STR][00:wc00] webu_parms_edit: camid: >< thread: >-1< cmd1: >< cmd2: >< cmd3: ><
Aug 31 12:35:49 [DBG][ALL][00:wc00] webu_hostname: Full Host:  //192.168.33.43:8080
Aug 31 12:35:49 [INF][ALL][00:wc00] webu_client_connect: Connection from: 192.168.33.34
Aug 31 12:35:49 [DBG][ALL][00:wc00] webu_hostname: Full Host:  //192.168.33.43:8080
Aug 31 12:35:49 [DBG][STR][00:wc00] webu_answer_get: processing get
Aug 31 12:35:49 [INF][STR][00:wc00] webu_mhd_deinit: Closing connection
Aug 31 12:35:49 [DBG][STR][00:wc00] webu_parseurl: Sent url: /0/config.json
Aug 31 12:35:49 [DBG][STR][00:wc00] webu_parseurl: Decoded url: /0/config.json
Aug 31 12:35:49 [DBG][STR][00:wc00] webu_parms_edit: camid: >0< thread: >-1< cmd1: >config.json< cmd2: >< cmd3: ><
Aug 31 12:35:49 [DBG][ALL][00:wc00] webu_hostname: Full Host:  //192.168.33.43:8080
Aug 31 12:35:49 [DBG][ALL][00:wc00] webu_hostname: Above message repeats 1 times
Aug 31 12:35:49 [DBG][STR][00:wc00] webu_answer_get: processing get
Aug 31 12:35:49 [INF][STR][00:wc00] webu_mhd_deinit: Closing connection

`
